File: CSharp/SpaceVIL/SpaceVIL/Windows/DropDown.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Drawing;

namespace SpaceVIL
{
    abstract public class DropDown : DialogWindow
    {
        /// <summary>
        /// Constructs a DropDown
        /// </summary>
        public DropDown() { }

        public override void InitWindow()
        {
            WindowLayout Handler = new WindowLayout(this, "DropDown_" + GetCount(), "DropDown_" + GetCount());
            SetHandler(Handler);
            Handler.SetWindowTitle(DialogTitle);
            Handler.IsBorderHidden = true;
            Handler.IsAlwaysOnTop = true;
            Handler.IsCentered = false;
            Handler.IsResizeble = false;
        }
    }
}

File: Java/SpaceVIL/spacevil/src/main/java/com/spvessel/spacevil/DropDown.java
package com.spvessel.spacevil;

abstract public class DropDown extends DialogWindow {

    /**
     * Constructs a DropDown
     */
    public DropDown() {
    }

    @Override
    public void initWindow() {
        WindowLayout Handler = new WindowLayout("DropDown_" + getCount(), "DropDown_" + getCount());
        Handler.setWindowTitle(getDialogTitle());
        Handler.isBorderHidden = true;
        Handler.isAlwaysOnTop = true;
        Handler.isCentered = false;
        Handler.isResizable = false;
    }
}